native/SampleApps/RestAPIExplorer/RestAPIExplorer/bootconfig.plist

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-	<key>remoteAccessConsumerKey</key>
-	<string>3MVG98dostKihXN53TYStBIiS8NRTXcbDzn9nHPb3piMElfQDD.kTyHeXjKV9JNUbe5sZeSQ4CVY1Onzpq21N</string>
```

Comment:
Can we move the `consumerKey` and other attributes back to `bootconfig.plist` and load them like they used to be, instead of having them in `AppDelegate.swift`?